models/build.js

```diff
@@ -3,13 +3,13 @@
 const Joi = require('joi');
 const mutate = require('../lib/mutate');
 const Scm = require('../core/scm');
-const Regex = require('../config/regex');
 const Job = require('../config/job');
 const Step = require('./step');
 const ID = Joi.number().integer().positive();
 const PARENT_BUILD_ID = ID;
 const PARENT_BUILDS_ID = Joi.alternatives().try(ID, Joi.valid(null));
 const buildClusterName = Joi.reach(require('./buildCluster').base, 'name');
+const JOB_NAME_REGEX = /^(PR-[0-9]+:)?[\w-]+$/;
```

Comment:
might be better to move to https://github.com/screwdriver-cd/data-schema/blob/master/config/regex.js as `JOB_NAME_ALL` or something